.laminas-ci.json

```diff
@@ -1,10 +1,45 @@
 {
     "ignore_php_platform_requirements": {
-        "8.1": true
+        "8.0": false,
+        "8.1": false
     },
     "extensions": [
         "apcu"
     ],
+    "additional_checks": [
+        {
+            "name": "Unit tests with psr/cache v2",
+            "job": {
+                "php": "8.0",
+                "dependencies": "latest",
```

Comment:
should be possible to have `*` here so we can reduce the amount of entries to 2 here. 👍🏼

Reply:
Ah, we should also add `*` for `php` since we now dropped PHP 7.4 support and thus can test all versions

Reply:
fixed